### ISSUE TITLE
I don't get reactions via brid.gy from facebook back into my blog

### ISSUE THREAD

**Issue by matzoman** [idno/Facebook#36]
Is this related to pull request #32 and issue #31 ? 

What's the pending work in the known-facebook-plugin to make this work again?
Or are there some code changes in brid.gy right now missing to make this work again?
(if that's the case the reactions could be "collected" some day in the future...)

the discussion about the "user id fb bug" and "see original link" problems are quite hard to follow / understand...

**Comment by kylewm** [comment]
Yes, most likely related to those.

Long story short, Facebook broke the way we were embedding the original post URL in their metadata.

So now for Bridgy to match up the FB post with its Known original, we need to either:

1. include the original post URL in the body of the FB post.
2. construct the syndication URL that goes in setPosseLink, so that Bridgy can recognize it. (like `http://www.facebook.com/user.name/posts/123456789` instead of `http://www.facebook.com/10028438938/posts/123456789`)

**Comment by matzoman** [comment]
Thanks, now I understand.

(I just tried to change one posse url directly in the MySQL database in the metadata-table (_kids, don't try this at home!_) but somewhere it seemed to be cached...)

**Comment by kylewm** [comment]
I you'd have to change it in the entities table to change what is rendered. But yeah, manually modifying the database is... not fun :P

**Comment by kylewm** [comment]
Bridgy is actually doing some fancy footwork to match up fb user ids, so this *should* be working now with no additional work on Known's side, Yay!
